data/templates/ipsec/swanctl/peer.j2

```diff
@@ -32,6 +32,9 @@
 {% endif %}
 {% if peer_conf.force_udp_encapsulation is vyos_defined %}
         encap = yes
+{% endif %}
+{% if uniqreqids %}
```

Comment:
Any reason to not use the common pattern

```suggestion
{% if uniqreqids is vyos_defined %}
```

Which also tests if the supplied variable is not `None`?